src/SFA.DAS.EmployerAccounts.Web/Controllers/EmployerTeamController.cs

```diff
@@ -23,6 +23,9 @@ public class EmployerTeamController : BaseController
     {
         private readonly EmployerTeamOrchestrator _employerTeamOrchestrator;
         private readonly IAuthorizationService _authorizationService;
+        private const string Yes = "yes";
```

Comment:
These could be implemented as a Tristate enum.

Reply:
Done